srtcore/api.cpp

```diff
@@ -1039,9 +1039,15 @@ SRTSOCKET CUDTUnited::accept(const SRTSOCKET listen, sockaddr* pw_addr, int* pw_
       // Check if LISTENER has the SRTO_GROUPCONNECT flag set,
       // and the already accepted socket has successfully joined
       // the mirror group. If so, RETURN THE GROUP ID, not the socket ID.
-      if (ls->m_pUDT->m_bOPT_GroupConnect && s->m_IncludedGroup)
+      if (ls->m_pUDT->m_OPT_GroupConnect & 1 && s->m_IncludedGroup)
```

Comment:
Why unary operator? Is it still flag-based?

Reply:
Internally, yes. Although it's hard to say how it will expand in future. Trying to keep it in the form that would require little changes when its meaning is expanded in the future.

Reply:
ls->m_pUDT->m_OPT_GroupConnect == 1
or
ls->m_pUDT->m_OPT_GroupConnect != 0 
makes more sense to me.